projects/packages/protect-status/src/class-status.php

```diff
@@ -0,0 +1,311 @@
+<?php
```

Comment:
This file was copied over from the Protect plugin. No changes other than the 'use'  statements for `\Protect_Models\...`.
And also added the PACKAGE_VERSION, line 18.